src/cf-modals/src/cf-modals.less

```diff
@@ -0,0 +1,91 @@
+/* ==========================================================================
+   Capital Framework
+   Modal Styling
+   ========================================================================== */
+
+//
+// Theme variables
+//
+
+// .o-modal
+@modal-backdrop:   @black;
```

Comment:
should be gray-80 according to Lorelei's DM updates: https://ielerol.github.io/design-manual/page-components/modals.html#style

Reply:
Could we get screenshots comparing this difference? I discussed with Natalia and we're not sure which version is correct. @anselmbradford do you know any history for why the modals code is different than the spec?

Reply:
@ielerol I don't have any further history on this part, but here's a comparison of before/after. Let me know which to keep.

As is:
 ![screen shot 2017-09-12 at 1 21 26 am](https://user-images.githubusercontent.com/704760/30309342-e3f54684-9758-11e7-90bd-4d226fb64dbf.png)

Lighter backdrop:
![screen shot 2017-09-28 at 4 40 47 pm](https://user-images.githubusercontent.com/704760/30989487-e9fc794c-a46b-11e7-81bc-3e37550f96e0.png)

Reply:
Thanks @anselmbradford! We thinker the lighter example is working better, so let's go with the gray-80 and .7 opacity

Reply:
Done!